model-optimizer/extensions/ops/fake_output.py

```diff
@@ -0,0 +1,39 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from mo.front.common.partial_infer.elemental import copy_shape_infer, copy_value
+from mo.graph.graph import Graph
+from mo.middle.passes.infer import copy_type_infer
+from mo.ops.op import Op
+
+
+# This op is needed only to store the output name, it will be transformed into opset op and doing nothing
+class FakeOutput(Op):
+    op = 'FakeOutput'
+
+    def __init__(self, graph: Graph, attrs: dict):
+        super().__init__(graph, {
+            'op': self.op,
+            'type': None,
+            'version': None,
+
+            'infer': lambda n: copy_shape_infer(n, copy_value),
```

Comment:
The operation is replaced during the front phase so no need to specify `infer` and `type_infer` functions